.github/workflows/convert-to-json.yml

```diff
@@ -0,0 +1,38 @@
+name: Convert to JSON
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  convert-to-json:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Convert frequency-plans.yml to json
+      uses: fabasoad/yaml-json-xml-converter-action@v1.0.1
+      id: yam
+      with:
+        path: 'frequency-plans.yml'
+        from: 'yaml'
+        to: 'json'
+
+    - name: Save yaml2json result
+      shell: bash
+      run: echo "test" >> frequency-plans.json
```

Comment:
This is for testing still I suppose?

Reply:
Yeah this is just trying to get the flow to work. There's something else wrong with the yaml2json flow, running
`echo ${{ steps.yam.outputs.data }} > frequency-plans.json`
makes the gh action docker container crash with an ERROR 137 🤦🏼 https://github.com/TheThingsNetwork/lorawan-frequency-plans/runs/731493232?check_suite_focus=true#step:4:1

Trying to fix that now too.

Reply:
> This is for testing still I suppose?

works now, just needs permission to push